File: 448-find-all-numbers-disappeared-in-an-array/448-find-all-numbers-disappeared-in-an-array.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        res = []
        for item in nums:
            item = abs(item)
            if(nums[item-1] > 0):
                nums[item-1] *= -1
        for i in range(len(nums)):
            if(nums[i] > 0):
                res.append(i+1)
        
        return res
        

File: 448-find-all-numbers-disappeared-in-an-array/448-find-all-numbers-disappeared-in-an-array.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& nums) {
        vector<int>res;
        for(int i=0; i<nums.size(); i++)
        {
            int temp = abs(nums[i]); 
            if(nums[temp - 1] > 0)
                nums[temp - 1] *= -1;
        }
        
        for(int i=0; i<nums.size(); i++)
        {
            if(nums[i] > 0)
                res.push_back(i+1);
        }
        return res;
    }
};